src/NServiceBus.Core/Pipeline/CreatePipelineExecutionFunc.cs

```diff
@@ -24,9 +25,9 @@ static class BehaviorExtensions
         ///          context{N} => behavior{N}.Invoke(context{N},
         ///             context{N+1} => TaskEx.Completed))
         /// </code>
-        public static LambdaExpression CreatePipelineExecutionExpression(this IBehavior[] behaviors)
+        public static Delegate CreatePipelineExecutionExpression(this IBehavior[] behaviors, List<Expression> expressions = null)
```

Comment:
For approval testing purposes I added the List<Expression> overload. This allows me to intercept the expression and pretty print it